app/views/layouts/base.html.erb

```diff
@@ -117,7 +117,7 @@ See COPYRIGHT and LICENSE files for more details.
       <% if show_onboarding_modal? %>
         <section data-augmented-model-wrapper
                  data-modal-initialize-now="true"
-                 data-modal-class-name="onboarding-modal op-modal_autoheight -highlight">
+                 data-modal-class-name="onboarding-modal spot-modal_autoheight -highlight">
```

Comment:
The `-highlight` class can be removed. It is also still present in some other files.

Reply:
removed usage of `-highlight` and `spot-modal_autoheight`. Second is fine, as the standard modal now has a calculated height.